app/models/ont/run_factory.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+# Ont namespace
+module Ont
+  # RunFactory
+  # The factory will create runs from a list of libraries
+  class RunFactory
+    include ActiveModel::Model
+
+    validate :check_run, :check_flowcells
+
+    def initialize(library_names = [])
+      @flowcells = []
+      build_run(library_names)
```

Comment:
I think the RunFactory is going to be initialized with libraryNames and position, a FlowcellType input, which includes the position. So `[{position: 1, library_name: 'aName'}, {position: 2, library_name: 'anotherName'}]`

Reply:
Agreed. I thought that we would have a flowcell type with a position and a library name attribute.